### ISSUE TITLE
Can I Change my Username?

### ISSUE THREAD

**Issue by mattlovenuk** [keybase/client#20232]
I stupidly created my Keybase account while working at a company and I am now about to leave the company and go somewhere else. I referenced the company in my username, but I didn't realise how useful at the time Keybase would be, and I dont want to loose all my account info, contacts, lumens, not to mention I'm a "lucky airdropee". Is there any way that I can change my account name?

**Comment by ravimakhijapi** [comment]
I agree that usernames should be changeable and would also like to change mine to create more consistency across different online applications I use. 

I think public UUIDs or similar should be used to provide public-facing uniqueness to an account instead. The username change history can also be persisted perhaps to address potential misuse.
